src/Microsoft.Health.Dicom.Api/Controllers/RetrieveController.cs

```diff
@@ -152,6 +153,29 @@ public async Task<IActionResult> GetSeriesMetadataAsync([FromHeader(Name = IfNon
         return CreateResult(response);
     }
 
+    [Produces(KnownContentTypes.ImageJpeg)]
+    [ProducesResponseType((int)HttpStatusCode.OK)]
+    [ProducesResponseType(typeof(string), (int)HttpStatusCode.NoContent)]
+    [ProducesResponseType(typeof(string), (int)HttpStatusCode.BadRequest)]
+    [ProducesResponseType(typeof(string), (int)HttpStatusCode.NotFound)]
+    [ProducesResponseType(typeof(string), (int)HttpStatusCode.NotAcceptable)]
+    [HttpGet]
+    [VersionedPartitionRoute(KnownRoutes.InstanceRenderedRoute)]
+    [VersionedRoute(KnownRoutes.InstanceRenderedRoute)]
+    [AuditEventType(AuditEventSubType.RetrieveRendered)]
+    public async Task<IActionResult> GetRenderedInstanceAsync(
+        string studyInstanceUid,
+        string seriesInstanceUid,
+        string sopInstanceUid)
+    {
+        _logger.LogInformation("DICOM Web Retrieve Rendered Image Transaction request for instance received, for study: '{StudyInstanceUid}', series: '{SeriesInstanceUid}', instance: '{SopInstanceUid}'.", studyInstanceUid, seriesInstanceUid, sopInstanceUid);
```

Comment:
I think these are helpful for local development and testing since these dont get redacted locally- we have similar logs for all routes in the controller